plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/features/onboarding-tasks/task-lists.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Test the TaskLists class.
+ *
+ * @package WooCommerce\Admin\Tests\OnboardingTasks
+ */
+
+/**
+ * Test task fixture.
+ */
+require_once __DIR__ . '/test-task.php';
+
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\TaskList;
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\TaskLists;
+
+/**
+ * Class WC_Tests_OnboardingTasks_TaskLists
+ */
+class WC_Tests_OnboardingTasks_TaskLists extends WC_Unit_Test_Case {
+	/**
+	 * Setup test data. Called before every test.
+	 */
+	public function setUp(): void {
+		parent::setUp();
+		TaskLists::clear_lists();
+	}
+
+	/**
+	 * Tests that the "woocommerce_admin_experimental_onboarding_tasklists" filter is able to override tasklists.
```

Comment:
Nit: Since we don't actually replace task in this particular test
```suggestion
	 * Tests that the "woocommerce_admin_experimental_onboarding_tasklists" filter is able to append tasks to any tasklist.
```